src/features/metadata-instance-editor/Instance.js

```diff
@@ -219,19 +215,16 @@ class Instance extends React.PureComponent<Props, State> {
             isEditing: false,
             shouldShowCascadeOptions: false,
         });
-        onSave(
-            id,
-            this.createJSONPatch(currentData, originalData),
-            isCascadingPolicyApplicable
-                ? {
-                      canEdit: cascadePolicy ? cascadePolicy.canEdit : false,
-                      id: cascadePolicy ? cascadePolicy.id : undefined,
-                      isEnabled: isCascadingEnabled,
-                      overwrite: isCascadingOverwritten,
-                  }
-                : undefined,
-            cloneDeep(currentData),
-        );
+
+        const cascadePolicyData = {
+            ...cascadePolicy,
+            isEnabled: isFolderInstance ? isCascadingEnabled : false,
+            overwrite: isCascadingOverwritten,
+        };
+
+        const clonedCurrentData = cloneDeep(currentData);
+        const JSONPatch = this.createJSONPatch(clonedCurrentData, originalData);
+        onSave(id, JSONPatch, cascadePolicyData, clonedCurrentData);
```

Comment:
little confused on why this change is needed. in the original implementation, it was only passing `cascadePolicyData` when the policy was applicable. but now it's passing `cascadePolicy` in all cases?

Reply:
Relevant EUW service: https://git.dev.box.net/AppServices/end-user-web/blob/25e47e8a54c6a19f666d15e3ddf3ac09c158807a/src/server/controllers/metadata/saveInstances.js

please reference this as I think it will help understand what's going on here